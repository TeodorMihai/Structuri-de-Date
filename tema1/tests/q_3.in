1 10
::w 2 8 AAAA
1
::w 3 26 BBBB
1
::w 2 23 CCCC
1
::w 4 15 DDDD
1
::r 2 23
5
::w 7 9 EEEE
1
::w 7 19 FFFF
1
::r 2 23
1
::w 0 9 00FF
1
::w 8 726 AAAA
1
::w 2 0 0010
1
::w 5 106 0100
1
::w 1 1 ABCD
1
::w 9 2300 CBDA
1
::w 3 124 134
1
::w 0 9 00FF
20
::w 8 726 AAAA
1
::w 2 0 0010
1
::w 5 106 0100
1
::w 1 1 ABCD
1
::d 9 2300 
220
::w 9 2300 CBDA
1
::w 3 124 134
1
::w 0 9 00FF
1
::r 9 2300
1
::w 8 726 AAAA
1
::w 2 0 0010
1
::w 5 106 0100
1
::w 1 1 ABCD
1
::w 9 2300 CBDA
1
::w 3 124 134
1
::w 0 9 00FF
1
::w 8 726 AAAA
1
::w 2 0 0010
1
::w 5 106 0100
1
::w 1 1 ABCD
1
::w 9 2300 CBDA
1
::w 3 124 134
1
::w 0 9 00FF
1
::w 8 726 AAAA
1
::w 2 0 0010
1
::w 5 106 0100
1
::w 1 1 ABCD
1
::w 9 2300 CBDA
1
::w 3 124 134
1
::w 0 9 00FF
1
::w 8 726 AAAA
1
::w 2 0 0010
1
::w 5 106 0100
1
::w 1 1 ABCD
1
::w 9 2300 CBDA
1
::w 3 124 134
1
::w 0 9 00FF
10
::w 8 726 AAAA
1
::w 2 0 0010
1
::w 5 106 0100
1
::w 1 1 ABCD
1
::w 9 2300 CBDA
1
::w 3 124 134
1
::d 9 2300 
1
::d 8 726 
1
::d 5 106
25000
::r 5 107
25000 
::d 5 107
3
::e