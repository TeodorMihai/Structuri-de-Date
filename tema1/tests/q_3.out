CCCC
CCCC
33
CBDA
235
225
225
0000
14
7.31 7.03 7.01 7.01
